docs/metal.md

```diff
@@ -0,0 +1,164 @@
+# Bare Metal Support
+
+***Bare metal support is experimental, and may be removed at any time***
+
+## Introduction
+
+kOps has some experimental bare-metal support, specifically for nodes.  The idea
```

Comment:
worth of mentioning that control-plane (and other nodes) should be with public topology? If the topology is private, its going to be difficult to have working cluster internal network